### PR TITLE
Bradm/seo updates 2

### PR DESCRIPTION
**Pull request by bradmccarty**
More SEO updates, per our SEO/backlinks folks at Userp.

### CODE REVIEW

**Comment by mooreds** [comment]
@bradmccarty do you need someone to review?

File: astro/src/content/articles/identity-basics/magic-links.md
Comment (mark-robustelli) [inline astro/src/content/articles/identity-basics/magic-links.md:42]:
```suggestion
The comparison between the traditional login process using passwords and magic link authentication is both valid and pertinent. Let’s explore why magic links might be a superior choice for enhancing the authentication process.
```

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
Mostly comments and suggestions.  1 extra space that does need to be removed.

File: astro/src/content/articles/authentication/multi-factor-authentication.md
Comment (bradmccarty) [reply astro/src/content/articles/authentication/multi-factor-authentication.md:288]: This came from the SEO folks. I'm familiar, but I think this is google-centric.

File: astro/src/content/articles/authentication/multi-factor-authentication.md
Comment (bradmccarty) [reply astro/src/content/articles/authentication/multi-factor-authentication.md:10]: This was a suggestion from the SEO folks. My guess is that they saw some cannibalism in search performance away from this page.

**Comment by bradmccarty** [comment]
@mark-robustelli Addressed the changes, but it's still showing a blocked merge. Not sure why.

**Review by mark-robustelli — APPROVED** [review]

**Comment by mark-robustelli** [comment]
@bradmccarty try to merge now.  I had to approve after you made the change.

**Pull request merged**
